ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -342,6 +342,17 @@ EFVSC:
   implements:
   - OPERATIONAL
 
+FVSFC:
```

Comment:
please rename VSCF to follow convention.

Reply:
renamed to VSCF